front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx

```diff
@@ -33,7 +62,22 @@ const ScenarioDescription = ({
 
       <div className="scenario-description">
         {scenario.description && (
-          <div className="scenario-details-description">{scenario.description}</div>
+          <div
+            ref={descriptionRef}
+            title={isOpenedDescription ? '' : scenario.description}
+            className={`scenario-details-description ${isOpenedDescription ? 'opened' : 'not-opened'}`}
+            style={{ width: isOpenedDescription ? width + 120 : width }}
+          >
+            {scenario.description}
+            <span
+              onClick={showDescription}
+              role="button"
+              tabIndex={0}
+              className={isOpenedDescription ? 'displayed-description' : 'display-description'}
```

Comment:
There is already a class on the parent - do we need another one here?

Reply:
done